pkg/ccl/changefeedccl/sink_webhook_v2.go

```diff
@@ -224,7 +274,8 @@ func (sc *webhookSinkClient) Flush(ctx context.Context, batch SinkPayload) error
 	defer res.Body.Close()
 
 	if !(res.StatusCode >= http.StatusOK && res.StatusCode < http.StatusMultipleChoices) {
-		resBody, err := io.ReadAll(res.Body)
+		// response body may be compressed, so we need to use our reader with decompression support
```

Comment:
nit: here and elsewhere, please capitalize the first word of comments and end with a period

Reply:
done

Reply:
I think you may have missed this instance, though i see you fixed others.

Reply:
Do you mean I missed this one?
https://github.com/cockroachdb/cockroach/pull/138872/files#diff-8b79b659227dd56b0190adcc52faebe0116f8abed96e0284359e810b21f361b9R238

Reply:
Oh yes. I did